wgsl/index.bs

```diff
@@ -353,59 +353,102 @@ Note: Only a function declaration can contain other declarations.
 
 # Types # {#types}
 
-Note: For the syntax of declaring types in [SHORTNAME] please see the [[#grammar]].
-TODO(dneto): This note is probably editorially obsolete.
+Programs calculate values.
 
-Programs calculate values. Each value in [SHORTNAME] belongs to exactly one *type*.
-A type is a set of (mathematical) values.
+In [SHORTNAME], a <dfn noexport>type</dfn> is set of values, and each value belongs to exactly one type.
```

Comment:
Might be worth an example saying the signed integer, unsigned integer and floating point values of 1 are all different values.